presto-main/src/main/java/com/facebook/presto/operator/scalar/ArraySliceFunction.java

```diff
@@ -53,7 +53,7 @@ public static Block slice(
 
         long toIndex = Math.min(fromIndex + length, size + 1);
 
-        if (fromIndex >= toIndex || fromIndex < 0 || toIndex < 0) {
+        if (fromIndex >= toIndex || fromIndex < 1 || toIndex < 0) {
```

Comment:
Do we still need `toIndex < 0`? Because if we know `fromIndex < toIndex` and `fromIndex >=1`, then we know `toIndex > 1` for sure? ...